resharper/resharper-yaml/.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
```

Comment:
We already have one at the root of the main solution. There's a duplicate one here because this subfolder is also the citizenmatt/resharper-yaml repo (at least for now, so I can do a standalone R# plugin). There is a problem that I should have fixed before merging - the yaml code is formatted at 2 spaces, while the unity one is at 4 :) I'll do a separate PR fixing that sometime later.